docs/contributing/install_capi_operator.md

```diff
@@ -78,3 +78,31 @@ The cluster is now ready to install Rancher Turtles. The default behavior when i
 :::tip
 For more fine-grained control of the providers and other components installed with CAPI, see the [Add the infrastructure provider](../tasks/capi-operator/add_infrastructure_provider.md) section.
 :::
+''
+
+### Install Rancher Turtles without `Cluster API Operator` as a Helm dependency
```

Comment:
Not sure I am fully getting the idea of moving this section here: this is CAPI Operator installation documentation and we are adding **Turtles installation** _without CAPI Operator_ notes here? I think we can just remove these at all or if it can be still used for development purposes leave it where it was originally with "for dev purposes only" or similar notes

Reply:
We have some users who confused it with the production option, it might be a better UX to have all development options in one place and the other parts of the docs can focus on something users should consider for production.